app/jobs/message_relay_job.rb

```diff
@@ -4,12 +4,7 @@ class MessageRelayJob < ApplicationJob
   def perform(message)
     ActionCable
       .server
-      .broadcast "messages_for_#{message.room_id}", message: {
-        id: message.id,
-        content: message.content,
-        name: message.user.name,
-        created_at: message.created_at,
-        room_id: message.room_id
-      }
+      .broadcast "messages_for_#{message.room_id}",
```

Comment:
he won't be able to unless he's a varified user @pewniak747 
https://github.com/elharion/chatguru/blob/76dcbc5147233c015da74e2f5826eadd9e7083a1/app/channels/application_cable/connection.rb